src/log/parser.rs

```diff
@@ -43,14 +43,14 @@ impl std::fmt::Debug for ParsedLog {
         writeln!(f, "ParsedLog [")?;
         for (i, line) in self.0.iter().enumerate() {
             match line {
-                LogItem::Line(l)         => {
+                LogItem::Line(l) => {
                     let s = std::str::from_utf8(l).unwrap_or("ERROR UTF8 ENCODING");
                     writeln!(f, "[{i}] Line('{s}')")?
-                },
-                LogItem::Progress(u)     => writeln!(f, "[{i}] Progress({u})")?,
+                }
+                LogItem::Progress(u) => writeln!(f, "[{i}] Progress({u})")?,
                 LogItem::CurrentPhase(s) => writeln!(f, "[{i}] Phase({s})")?,
-                LogItem::State(Ok(_))    => writeln!(f, "[{i}] State::OK")?,
-                LogItem::State(Err(_))   => writeln!(f, "[{i}] State::Err")?,
+                LogItem::State(Ok(_)) => writeln!(f, "[{i}] State::OK")?,
+                LogItem::State(Err(_)) => writeln!(f, "[{i}] State::Err")?,
```

Comment:
I'm not a fan of such alignments so this is fine with me :)
(The alignments look pretty but I don't like their impact on diffs + they're annoying to maintain, unless it's automated by the IDE/editor/formatter)